woocommerce/wc-admin
# Setup Checklist: Remove Setup Menu Item and I'll Do This Later link

Fixes #4067

From feedback from @jameskoster and @pmcpinto - we have decided to remove the "Focus on the Setup Checklist" logic that was introduced in #3796 - so this branch removes the WooCommerce menu over-rides that hide all Woo menu items with a "Setup" menu item, and also removes the "I'll Do This Later" link under the setup checklist:

__Before__
<img width="1665" alt="before-task-list-focus" src="https://user-images.githubusercontent.com/22080/78814046-e2713200-7982-11ea-9669-4b016bd7afb6.png">

__After__
<img width="1665" alt="Dashboard ‹ WooCommerce ‹ My WordPress Site — WooCommerce 2020-04-08 10-22-17" src="https://user-images.githubusercontent.com/22080/78814059-e735e600-7982-11ea-81d9-9a0697e96208.png">

### Detailed test instructions:

- On a **new site** ( or setup your test site so the setup check list displays ), install WooCommerce 4.1 Beta
- Verify the screen looks like the above _before_ shot with Woo menu items hidden and "Setup" shown, and "I'll do this later" link shown.
- Install this branch as a plugin
- Verify the 'Setup' menu item is gone, as is the "I'll do this later" link



### Changelog Note:

Tweak: Remove Setup Menu
